README.md

```diff
@@ -113,7 +113,7 @@ each Portus version:
 | Portus | Docker Engine | Docker Distribution |
```

Comment:
Does it make sense to include Clair here?

Reply:
It does. Added 👍